src/ranged.cpp

```diff
@@ -1347,14 +1347,15 @@ static projectile make_gun_projectile( const item &gun ) {
     proj.range = gun.gun_range();
     proj.proj_effects = gun.ammo_effects();
 
+    const auto atype = gun.ammo_data();
```

Comment:
`ammo_type` is just 4 characters more, but would make the code more descriptive.